tests/cypress/support/index.js

```diff
@@ -14,6 +14,7 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
+import '@babel/polyfill';
```

Comment:
The server build gets polyfills from the renderer, the standalone build adds it to the entrypoint. Unit tests get it from cypress, here.